backend/packages/Upgrade/src/api/repositories/QueryRepository.ts

```diff
@@ -36,6 +36,26 @@ export class QueryRepository extends Repository<Query> {
     return result.raw[0];
   }
 
+  public async insertQueries(queryDoc: Array<Partial<Query>>, entityManager: EntityManager): Promise<Query> {
```

Comment:
Why haven't you used `upsertQuery`?

Reply:
As in upsert query it will check for conflict each time in db through all queries and its not required for inserting fresh queries as it wont be present in db. I think why to add more computation on db. Even for conditions and partitions we have separate queries for upsert and insert.